src/pdfviewer/PDFDocument.cpp

```diff
@@ -3898,6 +3913,7 @@ bool PDFDocument::closeElement()
 	else if (dwOverview && dwOverview->isVisible()) dwOverview->hide();
 	else if (configManager->useEscForClosingEmbeddedViewer && isVisible()) {
 		// Note: avoid crash on osx where esc key is passed to hidden window
+		toggleFullScreen(false);
```

Comment:
what scenario is fixed here ?

Reply:
before closing the window we need to undo the changes done for fullscreen window mode.

Reply:
see 3rd list item of PR